omod/src/main/webapp/datasets/datasetEditor.jsp

```diff
@@ -135,6 +164,21 @@
 					</ul>
 				</td>
 			</tr>
+            <tr>
+	            <td>
+		            <c:if test="${!empty dataSetDefinition.name}">
```

Comment:
Whats wrong with git/me! the indention looks very bad here compared to what I have locally. 

cc: @dkayiwa

Reply:
It must be to do with your IDE

Reply:
I don't think so cause I have tried it with other Text Editors like sublime and Atom and looks fine. @dkayiwa 

But I have manually edited it using gist. Gonna use this https://gist.github.com/samuelmale/8eb629ac962e0af9d55fc61021880b0d